.github/workflows/tests.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - master
   pull_request:
+  workflow_dispatch:
```

Comment:
```suggestion
```

Reply:
Agreed. It is a leftover from my setup to run the workflows.